From 9ff62052a9490c96366d97096a039dca30702b2d Mon Sep 17 00:00:00 2001
From: Sergio Prado <sergio.prado@e-labworks.com>
Date: Fri, 3 Jul 2020 21:09:20 -0300
Subject: [PATCH] Fix redirection of errno header

The POSIX correct header to include is <errno.h>.

The glibc silently redirects incorrect includes, but musl libc prints
a warning:
   #warning redirecting incorrect #include <sys/errno.h> to <errno.h>

Signed-off-by: Sergio Prado <sergio.prado@e-labworks.com>
---
 c-utility/pal/linux/socket_async_os.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/c-utility/pal/linux/socket_async_os.h b/c-utility/pal/linux/socket_async_os.h
index bd59e487c099..1797089851e3 100644
--- a/c-utility/pal/linux/socket_async_os.h
+++ b/c-utility/pal/linux/socket_async_os.h
@@ -10,7 +10,7 @@
 #include <sys/select.h>
 #include <netinet/in.h>
 #include <netinet/tcp.h>
-#include <sys/errno.h>
+#include <errno.h>
 #include <fcntl.h>
 #include <unistd.h>
 #include <netdb.h>
-- 
2.17.1

